matic/migrations-graph-testing/5_test_transactions.js

```diff
@@ -12,66 +12,126 @@ const twentyPercentMonthlyHarbergerTax = "240" + "0000000000"; // Harberger tax
+
+      await steward.buyAuction("4", ether("65"), 50000, ether("1000"), {
+        from: accounts[3],
       });
 
-      await paymentToken.mint(accounts[2], ether("100000"));
-      await paymentToken.approve(steward.address, ether("50000000"), {
-        from: accounts[2],
+      await steward.buyAuction("5", ether("70"), 50000, ether("1000"), {
+        from: accounts[4],
       });
-      await steward.buyAuction("3", ether("60"), 50000, ether("5050"), {
-        from: accounts[2],
+
+      await steward.buyAuction("6", ether("75"), 50000, ether("1000"), {
+        from: accounts[5],
       });
 
-      await paymentToken.mint(accounts[3], ether("100000"));
-      await paymentToken.approve(steward.address, ether("50000000"), {
-        from: accounts[3],
+      console.log("..advance time by one week");
+      await time.increase("604800");
+
+      console.log(
+        "Acc6 buys all of Acc1's tokens for 10 eth more then their original price"
+      );
+
+      await steward.buy("1", ether("60"), ether("50"), 50000, ether("1000"), {
+        from: accounts[6],
       });
-      await steward.buyAuction("4", ether("65"), 50000, ether("8000"), {
-        from: accounts[3],
+
+      await steward.buy("2", ether("65"), ether("55"), 50000, ether("1000"), {
+        from: accounts[6],
       });
 
-      // User 2 adds more deposit
-      await steward.depositWei(ether("68"), {
-        from: accounts[2],
+      await steward.buy("3", ether("70"),ether("60"), 50000, ether("1000"), {
+        from: accounts[6],
       });
 
-      // User 3 buys user 1's token
-      await steward.buy("2", ether("88"), ether("55"), 50000, ether("456"), {
+      console.log("..advance time by one week");
+      await time.increase("604800");
+
+      console.log(
+        "Acc1 buys token 5 from acc 4 for 100eth where prevPrice was 70eth"
+      );
+    
+      await steward.buy("5", ether("100"), ether("70"), 50000, ether("1000"), {
+        from: accounts[1],
+      });
+
+      console.log("..advance time by one week");
+      await time.increase("604800");
+
+      console.log("Acct 3 adds more deposit");
+
+      await steward.depositWei(ether("50"), {
         from: accounts[3],
       });
+
+      //// All orgs and patrons withdraw all //// 
+
+      console.log("All orgs and patrons withdraw all"); 
+
+      let org1Address = "0x707c0041f6e87411812f9e98fd99c9eddfd0b2a0";
+      let org2Address = "0x2b48B87B7d168D0a8b7e1526ff90e10876E46067";
+      // let org1Balance = await steward.BenefactorFunds()
+      // let org2Balance = await steward.BenefactorFunds()
+      console.log(steward.benefactorFunds(org1Address).toString());
+      
+      // await steward.withdrawBenefactorFunds(ether(org1Balance.toString()), {
+      //   from: org1Address,
+      // });
+
+      // await steward.withdrawBenefactorFunds(ether(org2Balance.toString()), {
+      //   from: org2Address,
+      // });
+
+      // iterate through all user accts and withdrawDeposit 
```

Comment:
this isn't working for some accounts/users. says 'withdrawing too much' will figure out

Reply:
You can use the 'exit' function rather. This is due to how time progresses in smart contracts